docs/gui/tooltips.rst

```diff
@@ -0,0 +1,15 @@
+gui/tooltips
+============
+
+.. dfhack-tool::
+    :summary: Show tooltips with useful info.
+    :tags: fort inspection
+
+This script shows "tooltips" following units and/or mouse with job names.
```

Comment:
needs more description of the two options and their effects.

Reply:
add a bit of clarification...
It's kind of hard to describe though